composer.json

```diff
@@ -73,7 +73,10 @@
             "@process-replacements",
             "rm -rf tests/drupal-project .travis.yml"
         ],
-        "post-create-project-cmd": "@init-setup",
+        "post-create-project-cmd": [
+            "@init-setup",
+            "rm -rf ./.github/workflows/test-drupal-setup.yml"
```

Comment:
If I understand this correctly this workflow is a test for this project, but when we're setting up new projects, this should be removed - is not needed anymore.